### ISSUE TITLE
API of transformer 3D image

### ISSUE THREAD

**Issue by Barnonewdm** [matthew-brett/transforms3d#26]
Is there any API for transformer 3D image?

**Comment by matthew-brett** [comment]
No, not really, but this library is for generating transforms.  I would use `scipy.ndimage` to apply them - have a look at https://bic-berkeley.github.io/psych-214-fall-2016/resampling_with_ndimage.html
